test/volumes/cortx-pv-shared.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: cortx-pv-shared
+spec:
+  storageClassName: shared-pv
+  capacity:
+    storage: 20Gi
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Filesystem
+  hostPath:
+    path: /data/pv/shared
```

Comment:
this path needs to be replaced from config dynamically, before PV/PVC can be deployed